bin/impl/fluo_deploy/main.py

```diff
@@ -93,6 +93,12 @@ def launch_cluster(conn, config):
       exit('ERROR - Image not found for instance type: '+instance_type)
 
     bdm = BlockDeviceMapping()
+    if config.ebs_root_size() > 0 :
+      bdt = BlockDeviceType()
```

Comment:
This code worked w/ m1.large.   m3.xlarge may require setting related to ebs optimized, not sure.